packages/ecc-client-ga4gh-tes/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "ecc-client-ga4gh-tes",
+  "description": "Web components for TES",
+  "main": "index.js",
+  "private": true,
+  "type": "module",
+  "version": "0.1.0",
+  "author": "",
+  "license": "ISC",
+  "repository": "",
+  "scripts": {
+    "dev": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\"",
```

Comment:
done, I purposely removed wds because it would start the server, and I had the start script for that. I have added it though, Ill remove start script .